ToDo/src/main/java/com/todo/demo/Note/Note.java

```diff
@@ -0,0 +1,58 @@
+package com.todo.demo.Note;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import java.util.Date;
+
+//Mongo database annotation -- Model Class
+@Document("Note")
+public class Note {
```

Comment:
use lombok, check lombok on internet and check @Data, @AllargsConstructor

Reply:
Checked and updated.